packages/repository/src/repositories/kv.repository.bridge.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import * as legacy from 'loopback-datasource-juggler';
+
+import {Options, DataObject} from '../common-types';
+import {Model} from '../model';
+
+import {KVRepository, AsyncIterator, KVFilter} from './kv.repository';
+
+import {juggler, ensurePromise} from './legacy-juggler-bridge';
+
+/**
+ * Polyfill for Symbol.asyncIterator
+ * See https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-3.html
+ */
+// tslint:disable-next-line:no-any
+if (!(Symbol as any).asyncIterator) {
+  // tslint:disable-next-line:no-any
+  (Symbol as any).asyncIterator = Symbol.for('Symbol.asyncIterator');
+}
+
+/**
+ * An implementation of KVRepository based on loopback-datasource-juggler
+ */
+export class DefaultKVRepository<T extends Model> implements KVRepository<T> {
```

Comment:
While you are touching these APIs, could you please rename `KV` to `KeyValue` for more clarity? `DefaultKeyValueRepository` and `KeyValueRepository`.

Reply:
Renamed